src/twfy_tools/common/mailchimp.py

```diff
@@ -0,0 +1,686 @@
+import datetime
```

Comment:
Sorry, this is quite a big file and lots of it doesn't appear to be used by this PR. Do you want me to review the whole file? Seems to be quite a lot of repetitive boilerplate, e.g. list_name_to_unique_id/ segment_name_to_unique_id/ template_name_to_unique_id all seem to do the same thing, just on a different thing; I don't see why MailchimpHandler has to call top-level functions of exactly the same name with all the arguments having to be repeated, that kind of thing. I'm sure it's all fine 🤷

Reply:
Sorry yeah, it's a general helper file I've built up for projects that use mailchimp - the duplication was for backwards compatibility. Could be simplified for this but I do know it at least works in this form.